common/lib/xmodule/xmodule/course_module.py

```diff
@@ -35,6 +35,13 @@
 CATALOG_VISIBILITY_ABOUT = "about"
 CATALOG_VISIBILITY_NONE = "none"
 
+DEFAULT_COURSE_VISIBILITY_IN_CATALOG =\
```

Comment:
Please move `getattr` to this top line as we don't like to end our lines with a trailing slash.  Thanks.